cognite/client/data_classes/filters.py

```diff
@@ -262,7 +262,8 @@ class And(CompoundFilter):
         Combine an In and an Equals filter::
 
             >>> from cognite.client.data_classes.filters import And, Equals, In
-            >>> filter = And(Equals(("some", "property"), 42), In(("another", "property"), ["a", "b", "c"]))
+            >>> myView = client.data_modeling.views.retrieve(ids=("space", "external_id", "version"))
```

Comment:
This returns a `ViewList`, so you either need the hacky comma unpacking:
`>>> myView, = client.data_...`

or add `[0]`:
`>>> ...retrieve(ids=("space", "external_id", "version"))[0]`